share/rocm/cmake/header_template.h.in

```diff
@@ -11,7 +11,7 @@
 #if defined(_MSC_VER)
 #pragma message(": warning:This file is deprecated. Use the header file from @header_location@ by using #include <@correct_include@>")
 #elif defined(__GNUC__)
-#warning "This file is deprecated. Use the header file from @header_location@ by using #include <@correct_include@>"
+#pragma message(": warning : This file is deprecated. Use the header file from @header_location@ by using #include <@correct_include@>")
```

Comment:
The `: ` at the beginning of the message was primarily intended to make the formatting nicer when using MSC. I haven't checked the formatting here, but it's worth doing.

Reply:
I had the same thought, but I ran the tests when helping @nunnikri fix the failures and it seemed ok to me.